readme.md

```diff
@@ -23,19 +23,20 @@
 
 React is a library used to craft modern day UI and create views for the front-end in web, client and native applications.
 
-> **Selling Point:** By modeling small compatible components that focus on just rendering a view, we as developers can move business logic out of the DOM, and therefore improve our app's performance, maintainability, modularity, and readability.
+> **Selling Point:** By modeling small compatible components that focus on just rendering a view, we as developers can move business logic out of the DOM, and therefore improve our app's performance, maintainability, modularity and readability.
 
 #### Some History
 
 The first thing most people hear about React is "Facebook uses it."
-* First used by Facebook in 2011. Then Instagram in 2012.
+* First used by Facebook in 2011.
+* Then Instagram in 2012.
 * Went open source in May 2013.
 
-React was born out of Facebook's frustration with the traditional MVC model and how...
+React was born out of Facebook's frustration with the traditional MVC model and how..
   * Re-rendering something meant re-rendering everything (or just a lot).
   * That had negative implications on processing power and ultimately user experience, which at times became glitchy and laggy.
 
-> If you want to get a taste of what React's all about, [here's an introduction from React.js Conf 2015](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510). Recommend starting around the 8:35 mark and watching until 16:30.
+> If you want to get a taste of what React's all about, [here's an introduction from the 2015 React.js Conference](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510). Recommend starting around the 8:35 mark and watching until 16:30.
```

Comment:
`React.js Conf 2015` is the official name, at least according to their YouTube channel: https://www.youtube.com/playlist?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr